### DISCUSSION TITLE
model-viewer: "version": "2.0.1",

### DISCUSSION THREAD

**Discussion by NipunGoelFAI** [google/model-viewer#4458] [Q&A]
### Description

The attached glb is opening very well in the AR mode within the Android mobile, whereas in iOS in View in AR mode the 3D model opens in enlarged mode because of which 3D model goes outside the mobile screen and then when we try to reduce the size in mobile using two fingers, then after lot of tries its size come within the mobile screen.

#### Live Demo

Attached Recording from iPhone

### Version

- model-viewer: v2.0.1

### Browser Affected

- [ ] Chrome, version: xx.x.xxxx.xx
- [ ] Edge
- [ ] Firefox
- [ ] IE
- [ x] Safari

### OS

- [ ] Android
- [x ] iOS
- [ ] Linux
- [ ] MacOS
- [ ] Windows

### AR

- [ ] WebXR
- [ ] SceneViewer
- [x ] QuickLook

**Comment by NipunGoelFAI** [comment]
https://github.com/google/model-viewer/assets/108470776/14d9acaf-9020-414f-9e99-abef84133ea3

**Comment by elalish** [comment]
Does it repro with the latest MV (v3.2.1)?

**Reply by NipunGoelFAI** [reply]
We tried using the following way in our code, which will ensure that the latest version of model-viewer gets used and even after that the issue is reproducible:

<script type="module" src="https://unpkg.com/@google/model-viewer/dist/model-viewer.min.js"></script>

**Reply by elalish** [reply]
It looks like your model is in the wrong units: glTF is always in meters, so a common mistake is to model in cm and not apply a scale conversion on export. You can fix this with https://modelviewer.dev/docs/#entrydocs-scenegraph-attributes-scale

It should have the same problem on Android - it may have just not found the floor yet.

**Reply by NipunGoelFAI** [reply]
Let me try and get back.

**Reply by NipunGoelFAI** [reply]
We tried creating model with the suggestions given, but still we are facing the same issue. Attaching herewith the models both usdz and glb. Also, attaching herewith the screenshot from blender with the settings. 
![scene_properties](https://github.com/google/model-viewer/assets/108470776/f5f6f439-ba25-490b-8868-a534555f6880)
![STL_settings](https://github.com/google/model-viewer/assets/108470776/f7101f01-6232-44f7-9216-3d0ab932e8cc)

**Reply by elalish** [reply]
Open your model in our [editor](https://modelviewer.dev/editor/), then click the button under "Validation Report". Post here the width, height, and length it reports.

**Reply by NipunGoelFAI** [reply]
![image](https://github.com/google/model-viewer/assets/108470776/0d1394ce-a655-40c6-accb-49a6f2a861e0)

**Reply by NipunGoelFAI** [reply]
Any updates? Still waiting for the response after my update.

**Reply by elalish** [reply]
Yes, as I guessed, this is working as intended. You are trying to place a 5.5 m tall (~18 feet) Santa on your keyboard. If you point your phone straight up you might see his legs towering above you. I'd recommend `scale="0.1 0.1 0.1"`.

**Reply by NipunGoelFAI** [reply]
Thanks @elalish , the suggestion really helped us. The issue got resolved.

**Comment by NipunGoelFAI** [comment]
Thanks @elalish , the suggestion really helped us. The issue got resolved.
